packages/esm-patient-registration-app/src/client-registry/hie-client-registry/modal/confirm-hie.modal.tsx

```diff
@@ -1,20 +1,62 @@
 import React from 'react';
 import { useTranslation } from 'react-i18next';
 import { Button, ModalBody, ModalHeader, ModalFooter, Accordion, AccordionItem, CodeSnippet } from '@carbon/react';
-import styles from './confirm-hie.scss';
 import { age, ExtensionSlot, formatDate } from '@openmrs/esm-framework';
 import { type HIEPatient } from '../hie-types';
 import capitalize from 'lodash-es/capitalize';
+import styles from './confirm-hie.scss';
 
 const PatientInfo: React.FC<{ label: string; value: string }> = ({ label, value }) => {
   return (
-    <div style={{ display: 'grid', gridTemplateColumns: '0.25fr 0.75fr', margin: '0.25rem' }}>
-      <span style={{ minWidth: '5rem', fontWeight: 'bold' }}>{label}</span>
-      <span>{value}</span>
+    <div className={styles.patientInfo}>
+      <span className={styles.patientInfoLabel}>{label}</span>
+      <span>{value || '--'}</span>
     </div>
   );
 };
```

Comment:
It would be better to have this in a separate file as well

Reply:
sorted